qutebrowser/utils/urlutils.py

```diff
@@ -298,14 +298,28 @@ def qurl_from_user_input(urlstr):
     else:
         ipstr = urlstr.strip()
         rest = ''
+
+    # This may error out because we first use this function to validate the
+    # options (_validate_all) before the register the config...
+    try:
+        default_scheme = config.get('network', 'scheme-order')[0]
+    except KeyError:
+        default_scheme = None
```

Comment:
From a style point, I agree - however, if for some reason (say, a bug) `config` wouldn't be available here anymore, scheme order would just silently stop working instead of raising the exception - which isn't really a good thing. What do you think?